sdk/cmake/CMakeMacros.cmake

```diff
@@ -642,6 +642,7 @@ function(set_module_type MODULE TYPE)
     endif()
 
     # Set base address
+    # Use 'IMAGEBASE default' to skip these set_image_base(), especially for win32dll test files
     if(__module_IMAGEBASE)
```

Comment:
Not sure, then, why they are removed in the cmakelists above :thinking:

Reply:
Their type is `module`, not `win32dll`.